src/features/chat/components/AudioPlayer.tsx

```
@@ -218,17 +248,20 @@ const AudioPlayer: React.FC<AudioPlayerProps> = React.memo(({ src, variant, comp
   const iconScale = compact ? 'w-4 h-4' : 'w-5 h-5';
   const barHeight = compact ? 'h-6' : 'h-8';
   const containerPad = compact ? 'p-1.5 gap-2' : 'p-2 gap-2.5';
+  const currentTimeLabel = resolvedSrc ? formatTime(currentTime) : '0:00';
+  const durationLabel = resolvedSrc ? (isLoaded ? formatTime(duration) : '--:--') : (statusText || '--:--');
 
   return (
     <div className={`flex items-center ${containerPad} ${containerClasses} w-full select-none sketchy-border-thin`}>
-      <audio ref={audioRef} src={src} preload="metadata" />
+      {resolvedSrc ? <audio ref={audioRef} src={resolvedSrc} preload="metadata" /> : null}
```

Biome (2.4.6)

[error] Provide a track for captions when using audio or video elements.

(lint/a11y/useMediaCaption)